src/ralph/ui/static/ui/js/app/data_center/controllers.js

```diff
@@ -28,6 +28,10 @@ angular
         };
 
         $scope.$on('edit_rack', function (event, rack) {
+            angular.forEach($scope.data_center.rack_set, function(rack) {
+                rack.active = false;
+            });
+            rack.active = true;
```

Comment:
there is no way to active will be false. If previous loop set it like a false, here you back to true.